daemon/src/GHCSpecter/Util/Timing.hs

```diff
@@ -8,23 +8,32 @@ module GHCSpecter.Util.Timing
 
     -- * timing info utilities
     isInProgress,
+    isCompiled,
```

Comment:
isInProgress is rather confusing. We had better `isTimeInTimerRange` or something like that.

Reply:
Done